dolly-backend-domain/src/main/java/no/nav/dolly/domain/resultset/tpsf/Statsborgerskap.java

```diff
@@ -20,4 +20,5 @@
     private Long id;
     private String statsborgerskap;
     private LocalDateTime statsborgerskapRegdato;
```

Comment:
Ja synes dette er en grei tilnærming. Egentlig er regdato tidspunkt for registering som kan være noe annet enn fradato, men vi har ikke skilt på disse og her blir regdato brukt som fradato. Hvis vi dette ble laget som nytt burde det hete fradato som matcher tildato. :)

Reply:
Ja enig der, tenkte registreringsdato skulle få lov til å stå 😄

Reply:
💯